fe/fe-core/src/test/java/com/starrocks/transaction/ConcurrentTxnTest.java

```diff
@@ -236,9 +59,10 @@ public void testConcurrentLoad() throws Exception {
                     }
                 }
             }
-            System.out.printf("tableRead: %d scanQueryId: %d\n", totalTableRead.get(), PseudoBackend.scansByQueryId.size());
-            Assert.assertEquals(totalTableRead.get(), PseudoBackend.scansByQueryId.size());
-            Assert.assertEquals(totalTabletRead.get(), Tablet.getTotalReadSucceed());
+            System.out.printf("tableRead: %d scanQueryId: %d\n", DBLoad.TableLoad.totalTableRead.get(),
```

Comment:
Use log instead?

Reply:
That's just convenient to see output in Jenkins web ui